roles/network_plugin/calico_defaults/defaults/main.yml

```diff
@@ -22,7 +22,7 @@ calico_pool_blocksize: 26
 
 # Calico doesn't support ipip tunneling for the IPv6.
 calico_ipip_mode_ipv6: Never
-calico_vxlan_mode_ipv6: Never
+calico_vxlan_mode_ipv6: Always
```

Comment:
why?

Reply:
unification with ipv4, without this the tests will not work, there will be no connectivity between the pods